client/components/data/query-stored-cards/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isFetchingStoredCards } from 'state/stored-cards/selectors';
+import { fetchStoredCards } from 'state/stored-cards/actions';
+
+class QueryStoredCards extends Component {
+	componentWillMount() {
+		this.requestStoredCards();
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		if ( ! nextProps.isRequesting ) {
```

Comment:
We should probably check for this whenever we request for the list of stored cards.